tools/GENERATE-APG.sh

```diff
@@ -30,3 +30,16 @@ protoc ${SERVICE_PROTOS[*]} \
   	--go_cli_opt='root=apg' \
   	--go_cli_opt='gapic=github.com/apigee/registry/gapic' \
   	--go_cli_out='cmd/apg'
+
```

Comment:
If we're ok with this patch, we could go further and replace "APG_REGISTRY" with "REGISTRY", but that's a bigger change that we would need to propagate through the repo and any client configuration scripts.

Reply:
@shrutiparabgoogle I can imagine how that might happen in an authz configuration but it seems more likely (and better) for authz to give tokens varying sets of permissions. A more privileged token would have access to both services and a less-privileged one would only get to use the Registry service. On a much bigger scale, this is how Google auth tokens work - we get a token but it only works for the services we've enabled, and as we enable new services, that same token gets access to those new services.